docs/install_guides/pypi-openvino-dev.md

```diff
@@ -127,6 +127,27 @@ For example, to install and configure the components for working with TensorFlow
 
 For general troubleshooting steps and issues, see [Troubleshooting Guide for OpenVINO Installation](https://docs.openvino.ai/2022.2/openvino_docs_get_started_guide_troubleshooting.html). The following sections also provide explanations to several error messages. 
 
+### Errors with Installing via PIP for Users in China
+
+Users in China might encounter errors while downloading sources via PIP during OpenVINO™ installation. To resolve the issues, try one of the following options:
+   
+* Add the download source using the ``-i`` parameter with the Python ``pip`` command. For example: 
+
+   ``` sh
+   pip install openvino-dev -i https://mirrors.aliyun.com/pypi/simple/
+   ```
+   Use the ``--trusted-host`` parameter if the URL above is ``http`` instead of ``https``.
```

Comment:
```suggestion
```
The URL above is https, so it seems for this specific situation this is not relevant?

Reply:
Sorry I added this back, just in case the link changes in future or the user uses another source. I guess it won't hurt to have this reminder. :)